python/mxnet/numpy/fallback.py

```diff
@@ -114,11 +115,17 @@
 
 fallback_mod = sys.modules[__name__]
 
+def get_func(obj, doc):
+    """Get new numpy function with object and doc"""
+    @wraps(obj)
+    def wrapper(*args, **kwargs):
+        return obj(*args, **kwargs)
+    wrapper.__doc__ = doc
```

Comment:
`functools.wraps` would already copy the doc attribute. See https://docs.python.org/3.8/library/functools.html#functools.wraps

Reply:
Yes, `functools.wraps` will copy the doc attribute. But in the following for loop, some docs have been updated as `new_fn_doc`, we need to assign these new doc to MXNet numpy operator.